tutorial/simple-todos/03-forms-and-events.md

```diff
@@ -111,7 +111,8 @@ All that is left now is to make one final change: we need to show the newest tas
 ```html
 <script>
   ..
-  $m: tasks = TasksCollection.find({}, { sort: { createdAt: -1 } }).fetch()
+  let getTasks;
```

Comment:
This line isn't needed. Svelte (or in this case, `zodern:melte` since it is a tracker statement) will inject the variable declaration for you when compiling the file (https://svelte.dev/docs#component-format-script-2-assignments-are-reactive)